cmd/auth/logout.go

```diff
@@ -0,0 +1,139 @@
+package auth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+
+	"github.com/databricks/cli/libs/auth"
+	"github.com/databricks/cli/libs/auth/cache"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/databrickscfg"
+	"github.com/databricks/cli/libs/databrickscfg/profile"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+type LogoutSession struct {
```

Comment:
suggestion: I don't believe we intend for others to use this type. Does it need to be exported? None of its methods are exported (and for now I think it should remain that way).

Reply:
Implemented in commit bb35ca090f80e1246fe4f342443ea34a72596e1c